src/components/__tests__/HierarchicalMenu.js

```diff
@@ -435,10 +435,14 @@ describe('default render', () => {
       propsData: defaultProps,
     });
 
+    // wrapper.at(index) is deprecated at VTU@2.
+    // VTU@1: wrapper.at(index)
+    // VTU@2: wrapper[index]
+    // For simplicity, I've used `nth-child` modifier.
```

Comment:
comment isn't needed I think, maybe we can @deprecate or remove .at in our version of VTU1?

Reply:
What do you mean by removing .at? You mean at `test/utils` level? I tried to patch the `mount` function for VTU@1 but it can be 

- wrapper.at(index)
- wrapper.find(something).at(index)
- wrapper.find(something).findAll(else).at(index)

Unless we modify its prototype, it will be difficult.

Reply:
ok let's just rely on the test failing and remove the comment then